extensions/amp-viqeo-player/validator-amp-viqeo-player.protoascii

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-viqeo-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-viqeo-player"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-viqeo-player>
+  html_format: AMP
+  tag_name: "AMP-VIQEO"
+  requires_extension: "amp-viqeo-player"
+  attrs: {
+    name: "data-profileid"
```

Comment:
Optional and up to you, but if there are any constraints on the value of this or data-videoid, you can provide them here and the validator will assist developers in debugging issues by providing useful error messages.

In your example, it appears data-videoid is hex digits and data-profileid is a number. Something like `value_regex: "[0-9a-f]*"`.